### PR TITLE
use dsr ids to determine data source ids

### PR DESCRIPTION
**Pull request by TonioF**
Use the dsr ids provided by the Open Data Portal to identify data sets. This causes that more datasets are available and others, erroneously included, are removed.

### CODE REVIEW

File: cate/ds/esa_cci_odp.py [hunk 1]
```diff
@@ -741,45 +733,54 @@ async def _create_data_source(self, json_dict: dict, datasource_id: str):
...
+            return '5 days'
+        if time_value == '8-days':
+            return '8 days'
+        if time_value == '15-days':
+            return '15 days'
+        if time_value == '13-yrs':
+            return '13 years'
+        if time_value == 'climatology':
+            return 'climatology'
+        raise ValueError('Unknown time frequency format')
```
Comment (forman) [inline cate/ds/esa_cci_odp.py:783]: Convert the many `if`-s into simple lookup using a dict. 

B.t.w. why is this needed at all?

File: cate/ds/esa_cci_odp.py [hunk 2]
```diff
@@ -955,6 +953,27 @@ def __init__(self,
...
+
```
Comment (forman) [inline cate/ds/esa_cci_odp.py:976]: Code duplication. See above.

File: cate/ds/esa_cci_odp.py (hunk 1 above)
Comment (TonioF) [reply cate/ds/esa_cci_odp.py:783]: 1. Better readability. These will be part of the metdata
2. In this notation, they can be used in opensearch queries.

**Comment by TonioF** [comment]
Requested changes are implemented.
